pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -91,6 +91,8 @@ const (
 	affectLiveAndConfigLibvirtFlags = libvirt.DOMAIN_DEVICE_MODIFY_LIVE | libvirt.DOMAIN_DEVICE_MODIFY_CONFIG
 )
 
+const baseHotplugDiskDir = "/var/run/kubevirt/hotplug-disks/"
```

Comment:
@awels do you think value should be in the v1 schema?  That would look like: `v1.HotPlugDiskDir`

Reply:
Good question, might make sense if someone wants to find the disk image in the container (maybe from a side car or something).

Reply:
Its hard coded in the virt-launcher template, so no it would not change, so IMO it makes sense as a const somewhere. It is an internal implementation detail though, so from that perspective it doesn't make sense to make it part of the schema.

Reply:
I consider internal implementation detail to be apart of the schema.  For example these strings implement the DiskBus, so  `DiskBusUSB` can't be anything other than `usb`:
https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/schema.go#L626-L631
```
const (
	DiskBusSCSI   DiskBus = "scsi"
	DiskBusSATA   DiskBus = "sata"
	DiskBusVirtio DiskBus = "virtio"
	DiskBusUSB    DiskBus = "usb"
)
```

Reply:
Right but the bus is exposed in the API, you can specify the bus for a disk. The same cannot be said for the path that a hotplugged volume shows up in the virt-launcher pod. I don't have a strong opinion either way. It might be helpful for sidecars to know, but other than that I don't see how it would be useful for anyone but the implementation.

Reply:
If any kubevirt components are going to depend on the value, then I think we should attach it to a version to commit to it not changing.

Reply:
I declared this const in v1 schema for now and used that const everywhere else in code. If there is a better place to put it please let me know, I will move it there. Thanks for the discussion :slightly_smiling_face: